Sofa/Component/SolidMechanics/Spring/src/sofa/component/solidmechanics/spring/SpringForceField.inl

```diff
@@ -199,6 +267,15 @@ void SpringForceField<DataTypes>::initializeTopologyHandler(sofa::core::topology
                 msg_info(this) << "Removed points: [" << pointsRemoved->getArray() << "]";
                 applyRemovedPoints(pointsRemoved, mstateId);
             });
+
+        indices.linkToEdgeDataArray();  
```

Comment:
The scene: https://ci.inria.fr/sofa-ci-dev/job/sofa-framework/job/PR-3269/CI_CONFIG=fedora_clang,CI_PLUGINS=options,CI_TYPE=release/5/testReport/SceneTests/examples_Components_topology_TopologicalModifiers/RemovingPointSprings_scn/

is failing with this change because in this example the mesh are only points.
Before adding the callback you should add a test like this:
`if (topology->getTopologyType() == sofa::core::topology::TopologyElementType::EDGE)`